src/models/config.ts

```diff
@@ -0,0 +1,7 @@
+export interface VoyagerConfig {
+  showDataSourceSelector?: boolean;
```

Comment:
Thanks for explanation. This sounds reasonable.  Let's keep this part as is for now.